lockfile/src/lib.rs

```diff
@@ -161,9 +164,10 @@ impl Iterator for LockfileFormatIter {
             9 => LockfileFormat::NugetLock,
             10 => LockfileFormat::Msbuild,
             11 => LockfileFormat::Go,
-            12 => LockfileFormat::Cargo,
-            13 => LockfileFormat::Spdx,
-            14 => LockfileFormat::CycloneDX,
+            12 => LockfileFormat::GoMod,
```

Comment:
We prefer the old go format over the new one. Did you order it this way because of the failure chance?